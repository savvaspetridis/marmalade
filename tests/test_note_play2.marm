/* Testing play every type of note */

(play(), play(), play(), play(), play()) [44.w, 44.h, 44.q, 44.e, 44.s];