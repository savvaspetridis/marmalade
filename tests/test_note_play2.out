jMusic Play: Playing score One note score using JavaSound General MIDI soundbank.
jMusic Play: Waiting for the end of One note score.
jMusic MidiSynth: Stopped JavaSound MIDI playback
jMusic Play: Playing score One note score using JavaSound General MIDI soundbank.
jMusic Play: Waiting for the end of One note score.
jMusic MidiSynth: Stopped JavaSound MIDI playback
jMusic Play: Playing score One note score using JavaSound General MIDI soundbank.
jMusic Play: Waiting for the end of One note score.
jMusic MidiSynth: Stopped JavaSound MIDI playback
jMusic Play: Playing score One note score using JavaSound General MIDI soundbank.
jMusic Play: Waiting for the end of One note score.
jMusic MidiSynth: Stopped JavaSound MIDI playback
jMusic Play: Playing score One note score using JavaSound General MIDI soundbank.
jMusic Play: Waiting for the end of One note score.
jMusic MidiSynth: Stopped JavaSound MIDI playback